tests/integration/test_data/cached_dependencies.yaml

```diff
@@ -0,0 +1,20 @@
+# Test data for the Using cached dependencies test
+cached_dependencies:
```

Comment:
Do we still need this key here now this is the file name?

We could get rid of this key and be less verbose when acessing the data here!

Reply:
@athos-ribeiro I would not delete it, because my next PR will contain a new test. I would like to keep every test with special key.

Reply:
Ok, I decided to change it right now and change it next time as well. Who knows how long it will take to merge next request :D